includes/admin/post-types/meta-boxes/class-llms-meta-box-achievement-sync.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * LLMS_Meta_Box_Achievement_Sync class
+ *
+ * @package LifterLMS/Admin/PostTypes/MetaBoxes/Classes
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Meta box to sync between awarded achievements and achievement templates.
+ *
+ * @since [version]
+ */
+class LLMS_Meta_Box_Achievement_Sync extends LLMS_Abstract_Meta_Box_User_Engagement_Sync {
+
+	/**
+	 * Type of user engagement.
+	 *
+	 * @since [version]
+	 *
```

Comment:
From the [LifterLMS documentation standards](https://github.com/gocodebox/lifterlms/blob/trunk/docs/documentation-standards.md#tag-alignment-and-order):
> All changelog tags, `@since`, `@version`, and `@deprecated` should be grouped together with a space before the first `@since` tag and after the last tag in the group.

Reply:
yeah, you're right, as said before I was mislead by the example:
https://github.com/gocodebox/lifterlms/blob/trunk/docs/documentation-standards.md#class-members
But you're right.